gitops/snapshot_test.go

```diff
@@ -862,6 +863,8 @@ var _ = Describe("Gitops functions for managing Snapshots", Ordered, func() {
 				Expect(err).ToNot(HaveOccurred())
 				Expect(componentSnapshotInfos).To(HaveLen(2))
 				Expect(snapshot.Labels[gitops.SnapshotTypeLabel]).To(Equal("group"))
+				Expect(gitops.IsSnapshotCreatedByPACPushEvent(snapshot)).To(BeFalse())
+				Expect(!gitops.IsSnapshotCreatedByPACPushEvent(snapshot)).To(BeTrue())
```

Comment:
Aren't these two lines testing the same thing?

Should we create a new `It()` block and test the condition when the group snapshot is missing the `PipelineAsCodeEventTypeLabel` and the `IsSnapshotCreatedByPACPushEvent()` returns `false`

Reply:
Thanks for your question. Group snapshot has event-type: pull-request which is the same with the component snapshot which triggers the group snapshot creation. It is set before created. So I feel we don't need it. Do you agree?

Reply:
Thanks, that makes sense.
I do feel like the 2 assertion we're making above is redundant. Can we remove one of them?

Reply:
You are right. Let me remove one. Actually I wanted to test the multiple conditions in this functions.

Reply:
Thanks!